include/ipz_parser.hpp

```diff
@@ -42,6 +42,8 @@ class IpzVpdParser : public ParserInterface
         {
             m_vpdFileStream.open(vpdFilePath, std::ios::in | std::ios::out |
                                                   std::ios::binary);
+            m_vpdFileStream.exceptions(std::ifstream::badbit |
```

Comment:
This will not catch any exception if happens in file opening.
Ideally should come before open.

Reply:
ok